src/components/Table.tsx

```diff
@@ -0,0 +1,73 @@
+import {
+  useEffect,
+  useState,
+} from 'react';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+import { Loader } from './Loader';
+
+export const Table = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        setIsLoading(true);
+        const data = await getPeople();
+
+        setPeople(data);
+      } catch {
+        setIsError(true);
+      } finally {
+        setIsLoading(false);
+      }
+    };
+
+    fetchData();
+  }, []);
+
+  if (isLoading) {
+    return <Loader />;
+  }
+
+  return (
+    <>
```

Comment:
you can make if condition inside return method
```suggestion
  return (
    <>
      {isLoading ? <Loader /> : <>...</>}
    </>
  )
```